### DISCUSSION TITLE
Remove helicopter crash site's wreckage

### DISCUSSION THREAD

**Discussion by WyzVy** [TEHE-Studios/ExpandedHelicopterEvents#326] [Q&A]
One helicopter crashed in the middle of the road out of my base blocking it but I'm not able to dismantle it even with propane torch and welder mask, the option just dont pops up. I was wondering if there is a way to remove it as I dont seem to be able to find it. (Had to install and use cheat menu to remove the vehicle).

**Comment by Chuckleberry-Finn** [comment]
You can tow vehicles, dismantling is not part of the base mod, but there are a number of mods which let you dismantle vehicles.
